react-common/components/profile/UserPane.tsx

```diff
@@ -27,8 +29,13 @@ export const UserPane = (props: UserPaneProps) => {
 
     return <div className="profile-user-pane">
         <div className="profile-portrait">
-            { picture?.dataUrl ?
-                <img src={picture?.dataUrl} alt={pxt.U.lf("Profile Picture")} />
+            { picUrl ?
+                // Google user picture URL must have referrer policy set to no-referrer
+                // eslint-disable-next-line: react/no-danger
+                <div dangerouslySetInnerHTML={{__html: pxt.BrowserUtils.imgTag(picUrl, {
```

Comment:
Can we patch up the declaration of ImgHTMLAttributes to get around having to do this? e.g. throw this at the top of the file

```typescript
declare module 'react' {
    interface ImgHTMLAttributes<T> {
      referrerPolicy?:  "no-referrer" | "no-referrer-when-downgrade" | "origin" | "origin-when-cross-origin" | "unsafe-url";
    }
}
```

and the typing should work I believe (I just tested in a random tsx file)

<img width="811" alt="image" src="https://user-images.githubusercontent.com/5615930/185516060-30f07681-8557-4ea8-bbef-a2b8a90f09cc.png">

Benefit there is that it gets rid of runtime weirdness, just leaving compile time only weirdness

Reply:
Oh good suggestion, let me try that.